### PR TITLE
Batch operation rate limit

### PR DESCRIPTION
**Pull request by stephanos**
## What does this PR do?
Documents https://github.com/temporalio/cli/pull/366.

Specifically, the new `--rps` flag for most batch operations.

_Note that although `batch-reset` is a batch operation, it performs its work locally instead of on the server - so the flag does not apply here._

## Notes to reviewers

:warning: The CLI change has been merged but I'll have to _release_ the new CLI version before we can merge this, right?

### CODE REVIEW

File: docs-src/references/dynamic-configuration.md [hunk 1]
```diff
@@ -136,20 +136,22 @@ The Requests Per Second (RPS) dynamic configuration keys set the rate at which r
...
+| `matching.numTaskqueueWritePartitions` | Int | Number of write partitions for a Task Queue.                                                                                                                                                                                                       | 4             |
+| Worker                                 |     |                                                                                                                                                                                                                                                    |               |
+| `worker.batcherRPS`                    | Int | Rate limit for batch operations.                                                                                                                                                                                                                   |               |
```
Comment (stephanos) [inline docs-src/references/dynamic-configuration.md]: Added these two rows.

File: docs-src/references/dynamic-configuration.md [hunk 2]
```diff
@@ -150,6 +150,8 @@ Exceeding these limits results in `ResourceExhaustedError`.
...
 | `matching.numTaskqueueWritePartitions` | Int  | Number of write partitions for a Task Queue.                                                                                                                                                                                                         | 4             |
+| Worker                                 |      |                                                                                                                                                                                                                                                      |               |
+| `worker.batcherRPS`                    | Int  | Rate limit for batch operations.                                                                                                                                                                                                                     |               |
```
Comment (stephanos) [inline docs-src/references/dynamic-configuration.md:154]: Added these two rows.

**Review by rachfop — APPROVED** [review]
I'll merge this in tomorrow when the CLI is released!

**Comment by stephanos** [comment]
FYI I was told to wait with this until the `rps` feature is released in Server.

**Comment by MasonEgger** [comment]
This is now being handled by the auto-generation of CLI docs. Closing.

**Pull request closed without merging**